container/README.md

```diff
@@ -0,0 +1,45 @@
+# Agent Check: Container
+
+## Overview
+
+This check reports a set of metrics about any running containers, regardless of the runtime used to start them.
+
+## Setup
+
+### Installation
+
+Container is a core Agent 6 check and is automatically activated if any supported container runtime is detected.
+Configuring access to supported container runtimes (Docker, containerd) may be required depending on your environment.
+
+#### Installation on containers
+
+The `container` check requires some folders to be mounted to allow for automatic activation. This is handled by our official Helm Chart, the Datadog Operator, and as documented set ups for Kubernetes, Docker, ECS, and ECS Fargate.
+
+### Configuration
+
+Currently, the `container` check does not expose any specific configuration settings. To customize common fields or to force the activation of the `container` check, follow these steps:
+
+1. Create the `container.d/conf.yaml` file in the `conf.d/` folder at the root of your Agent's configuration directory.
+
+2. [Restart the Agent][2]
```

Comment:
Small nit, but it looks like `[2]` doesn't actually link to instructions on restarting the agent.